cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -837,6 +839,24 @@ export class Application {
         cy.wait(3 * SEC);
         this.closeApplicationDetails();
     }
+    validateOverrideAssessmentMessage(archetypes: Array<Archetype>): void {
```

Comment:
**suggesstion:** `Archetype[]` instead of `Array<Archetype>` It's literally the same but with shorter syntaxt

Reply:
done